process/sync/shardblock_test.go

```diff
@@ -1550,6 +1551,7 @@ func TestBootstrap_RollbackIsEmptyCallRollBackOneBlockToGenesisShouldWork(t *tes
 				//bytes represent a tx block body (strings are returns from txBlockUnit.Get which is also a stub here)
 				//copy only defined fields
 				obj = prevTxBlockBody
+				_ = obj
```

Comment:
this is masking an error
please do a deep copy

Reply:
done.